roles/pulp-api/templates/pulp-api.deployment.yaml.j2

```diff
@@ -64,6 +64,14 @@ spec:
         - name: assets-file-storage
           emptyDir: {}
 {% endif %}
+{% if db_fields_encryption_secret is defined %}
```

Comment:
Not saying this is wrong, but isn't this always going to be defined now? Either the user provides it or the code generates it.

Reply:
Just thinking about simplicity

Reply:
you are right, I'll change it